packages/SwingSet/src/kernel/state/storageWrapper.js

```diff
@@ -237,10 +234,10 @@ export function addHelpers(storage) {
 // write-back buffer wrapper (the CrankBuffer), but the keeper is unaware of
 // that.
 
-export function wrapStorage(hostStorage) {
-  const guardedHostStorage = guardStorage(hostStorage);
+export function wrapStorage(kvStore) {
```

Comment:
This is all correct, but it makes me wonder how exactly the `addTranscriptEntry` function will work: should it get held in the crank buffer (and thus this `wrapStorage` kinda wants to wrap both the `kvStore` and the `streamStore`? Or only in the block buffer?

We'll need to take a look at `kernel.js` or maybe `vatManager/transcript.js` where the entry gets added, and think about what the commit behavior should be.